libs/designsystem/src/lib/components/radio/radio.component.scss

```diff
@@ -41,6 +41,11 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
       margin-left: $spacing-to-edge;
       margin-right: $spacing-to-label;
     }
+
+    span {
+      align-self: center;
```

Comment:
Why is this necessary? 🤔

Reply:
https://github.com/kirbydesign/designsystem/pull/1374#discussion_r592188753

Reply:
resolved by: a8e1e31